web/src/copy_and_paste.js

```diff
@@ -294,29 +294,165 @@ function get_end_tr_from_endc($endc) {
     return $endc.parents(".selectable_row").first();
 }
 
+function cleanAttribute(attribute) {
+    // We replace any occurrences of one or more consecutive newlines followed by
+    // zero or more whitespace characters with a single newline character.
+    return attribute ? attribute.replaceAll(/(\n+\s*)+/g, "\n") : "";
+}
```

Comment:
Why is this called `cleanAttribute`? `deduplicate_newlines` seems like a clearer name.

Reply:
This is actually a function from the turndown library, so this is what it was named there -- Have renamed to `deduplicate_newlines` here now!